contracts/LifCrowdsale.sol

```diff
@@ -346,18 +357,56 @@ contract LifCrowdsale is Ownable, Pausable {
       foundationWallet.transfer(foundationBalanceCapWei);
 
       // TODO: create the MVM with a start block that equals one month after crowdsale ends
-      LifMarketValidationMechanism newMVM = new LifMarketValidationMechanism(
+      MVM = new LifMarketValidationMechanism(
         address(token), block.timestamp.add(10), 30 days, MVMPeriods, foundationWallet
       );
-      newMVM.fund.value(mmFundBalance)();
-      newMVM.transferOwnership(foundationWallet);
 
-      MVM = address(newMVM);
+      mintExtraTokens(uint256(MVMPeriods));
+
+      MVM.fund.value(mmFundBalance)();
+      MVM.transferOwnership(foundationWallet);
 
     }
   }
 
   /**
+     @dev Internal. Distribute the rest of the tokens among founders and advisors
```

Comment:
I'd explain what's the exact distribution, something like: "Distribute extra tokens among founders, team and the foundation long-term reserve. Founders receive 12.8% of tokens in a 4y (1y cliff) vesting schedule. Foundation long-term reserve receives 5% of tokens in a vesting schedule with the same duration as the MVM that starts when the MVM ends. An extra 7.2% is transferred to the foundation to be distributed among advisors and future hires"

Reply:
:+1: